docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -138,6 +139,37 @@ Type | Key | value | Description
 `ML_TRAIN_LAYER_TYPE_FC`    |                    |               | Fully connected layer
 &#xfeff;                    | unit               | (integer)     | number of outputs
 
+Since Tizen 6.5 and higher, each layer type has the following available properties:
+
+| Type                             | Key                         | value                | Description                                                           |
```

Comment:
Global for the table: 
1. Ensure the first letter of the sentences in the description is capital, to maintain consistency. (Description column)
2. Why some values are in brackets, while others are not, (Is there any specific reason?) (Value Column), If yes. then you can ignore this.
3. Please add period to all sentences in description column.

Reply:
> * Ensure the first letter of the sentences in the description is capital, to maintain consistency. (Description column)

It has consistency of, having `Type` description starting with capital, having `Key` description starting with non-capital on purpose. Could you state the reason to make all capital? https://github.com/Samsung/tizen-docs/pull/1418#discussion_r601041767

> * Why some values are in brackets, while others are not, (Is there any specific reason?) (Value Column), If yes. then you can ignore this.

Yes, `(foo)` states a type, `bar` states a raw string.

> * Please add period to all sentences in description column.

I checked that every descriptions are not sentences but phrases, plus, from 6.0, it was a convention to not put the period. Could you share the reasoning behind putting period to all the description?

Reply:
@zhoonit thanks for clarifying nicely. I agree with your answers and then the current text and formatting are correct.

Reply:
>  from 6.0, it was a convention to not put the period. Could you share the reasoning behind putting a period to all the descriptions?

If that was decided, then I have no issues following up with the same. Thanks for the clarity.